addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss

```diff
@@ -8,16 +8,15 @@
         display: none;
     }
     .close {
+        z-index: $zindex-modal;
         @include o-position-absolute(0, 0);
-        font-size: $font-size-sm;
-        width: 2 * $font-size-sm;
-        height: 2 * $font-size-sm;
+        height: $font-size-lg * 2;
+        width: $font-size-lg * 2;
```

Comment:
width before height to control my allergies please :p

Reply:
This mistake could have been avoided by using the size mixin 😜